azure-pipelines.yml

```diff
@@ -1,130 +1,73 @@
+  - job: Windows_DMD_latest
     timeoutInMinutes: 120
     pool:
       vmImage: 'vs2017-win2016'
     variables:
       VSINSTALLDIR: C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\
+      D_COMPILER: dmd
       HOST_DMD_VERSION: LATEST
     strategy:
       matrix:
         x64:
           OS: Win_64
           MODEL: 64
           ARCH: x64
-          D_COMPILER: dmd
-    steps:
-      - template: .azure-pipelines/windows.yml
-      - template: .azure-pipelines/windows-artifact.yml
-
-  - job: Windows_OMF_Bootstrap
-    timeoutInMinutes: 120
-    pool:
-      vmImage: 'vs2017-win2016'
-    variables:
-      VSINSTALLDIR: C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\
-      HOST_DMD_VERSION: 2.079.0
-    strategy:
-      matrix:
-        win32:
-          OS: Win_32
-          MODEL: 32
-          ARCH: x86
-          D_COMPILER: dmd
-    steps:
-      - template: .azure-pipelines/windows.yml
-
-  - job: Windows_OMF
-    timeoutInMinutes: 120
-    pool:
-      vmImage: 'vs2017-win2016'
-    variables:
-      VSINSTALLDIR: C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\
-      HOST_DMD_VERSION: LATEST
-    strategy:
-      matrix:
-        win32:
+        x86-OMF:
           OS: Win_32
           MODEL: 32
           ARCH: x86
-          D_COMPILER: dmd
     steps:
       - template: .azure-pipelines/windows.yml
       - template: .azure-pipelines/windows-artifact.yml
 
-  - job: Windows_VisualD
+  - job: Windows_VisualD_LDC
     timeoutInMinutes: 120
     pool:
       vmImage: 'vs2017-win2016'
     variables:
       VSINSTALLDIR: C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\
       HOST_DMD_VERSION: 2.090.0
+      D_COMPILER: ldc
+      VISUALD_VER: v0.49.0
+      LDC_VERSION: 1.20.0
     strategy:
       matrix:
-        win32-ldc:
-          OS: Win_32
-          MODEL: 32
-          ARCH: x86
-          D_COMPILER: ldc
-          VISUALD_VER: v0.49.0
-          LDC_VERSION: 1.20.0
-    steps:
-      - template: .azure-pipelines/windows-visual-studio.yml
-
-  - job: Windows_LDC_Debug
-    timeoutInMinutes: 120
-    pool:
-      vmImage: 'vs2017-win2016'
-    variables:
-      VSINSTALLDIR: C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\
-      HOST_DMD_VERSION: 2.090.0
-    strategy:
-      matrix:
-        x64-debug-ldc:
+        x64_Debug:
           OS: Win_64
           MODEL: 64
           ARCH: x64
           CONFIGURATION: Debug
-          D_COMPILER: ldc
-          VISUALD_VER: v0.49.0
-          LDC_VERSION: 1.20.0
-    steps:
-      - template: .azure-pipelines/windows-visual-studio.yml
-
-  - job: Windows_LDC_MinGW
-    timeoutInMinutes: 120
-    pool:
-      vmImage: 'vs2017-win2016'
-    variables:
-      VSINSTALLDIR: C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\
-      HOST_DMD_VERSION: 2.090.0
-    strategy:
-      matrix:
-        win32-ldc:
+        x86-mscoff:
+          OS: Win_32
+          MODEL: 32mscoff
```

Comment:
The `MODEL` for this job used to be `32` - but was overridden to `32mscoff` in https://github.com/dlang/dmd/blob/a0875a7e0bb9cd27f12b09eb96c391200a743034/.azure-pipelines/windows-msbuild.bat#L8-L11